_field-types/supported-field-types/star-tree.md

```diff
@@ -0,0 +1,199 @@
+
+## Prerequisites
+
+To use a star-tree index, follow the instructions in [Enabling a star-tree index]({{site.url}}{{site.baseurl}}/search-plugins/star-tree-index#enabling-a-star-tree-index).
+
+## Limitations
+
+The star-tree index feature has the following limitations:
+
+- A star-tree index should only be enabled on indexes whose data is not updated or deleted because standard updates and deletions are not accounted for in a star-tree index.
+- Currently, only `one` star-tree index can be created per index. Support for multiple star-trees will be added in a future version.
+
+## Examples
+
+The following examples show how to use a star-tree index.
+
+### Star-tree index mappings
+
+Define star-tree index mappings in the `composite` section in `mappings`. 
+
+The following example API request creates a corresponding star-tree index for all `request_aggs`. To compute metric aggregations for `request_size` and `latency` fields with queries on `port` and `status` fields, configure the following mappings:
+
+```json
+PUT logs
+{
+  "settings": {
+    "index.number_of_shards": 1,
+    "index.number_of_replicas": 0,
+    "index.composite_index": true
+  },
+  "mappings": {
+    "composite": {
+      "request_aggs": {
+        "type": "star_tree",
+        "config": {
+          "max_leaf_docs": 10000,
+          "skip_star_node_creation_for_dimensions": [
+            "port"
+          ],
+          "ordered_dimensions": [
+            {
+              "name": "status"
+            },
+            {
+              "name": "port"
+            }
+          ],
+          "metrics": [
+            {
+              "name": "request_size",
+              "stats": [
+                "sum",
+                "value_count",
+                "min",
+                "max"
+              ]
+            },
+            {
+              "name": "latency",
+              "stats": [
+                "sum",
+                "value_count",
+                "min",
+                "max"
+              ]
+            }
+          ]
+        }
+      }
+    },
+    "properties": {
+      "status": {
+        "type": "integer"
+      },
+      "port": {
+        "type": "integer"
+      },
+      "request_size": {
+        "type": "integer"
+      },
+      "latency": {
+        "type": "scaled_float",
+        "scaling_factor": 10
+      }
+    }
+  }
+}
+```
+
+
+
+## Star-tree mapping parameters
+
+Specify any star-tree configuration mapping options in the `config` section. Parameters cannot be modified without reindexing documents.
+
+The star-tree `config` section supports the following property.
+
+| Parameter  | Required/Optional | Description  | 
+| :--- | :--- | :--- |
+| `name` | Required | The name of the field. The field name should be present in the `properties` section as part of the index `mapping`. Ensure that the `doc_values` setting is `enabled` for any associated fields.
```

Comment:
Confused on this - `config` itself doesn't have a `name` property. Can we remove this ? 

Under config , user can specify `ordered_dimensions`, `metrics`, `max_leaf_docs` and `skip_star_node_creation_for_dimensions`.

Reply:
I'll remove this. We have the definitions for `max_leaf_docs` and `skip_star_node_creation_for_dimensions` on line 193.